control-plane/agents/Cargo.toml

```diff
@@ -37,8 +36,7 @@ tracing-futures = "0.2.4"
 rpc = "0.1.0"
 http = "0.2.3"
 paste = "1.0.4"
-store = { path = "../store" }
-types = { path = "../types"}
+control-plane-common = { path = "../../common" }
```

Comment:
should it just be called common?

Reply:
Done - renamed to common-lib to avoid name clash with agents/common.